autosklearn/ensemble_building/builder.py

```diff
@@ -437,7 +423,9 @@ def main(
 
         for run in requires_update:
             run.record_modified_times()  # So we don't count as modified next time
-            run.loss = self.loss(run, kind="ensemble")
+            run.losses = {
```

Comment:
What's the reason to calculate only one loss at a time?

Reply:
It meant less changes to the `loss` function and in general I prefer to loop to do the same things several times rather than have a function that does it all at once.

It also makes it easier to see what `run.losses` is rather than have it be the result of some function call.